### ISSUE TITLE
Request Flatpak package

### ISSUE THREAD

**Issue by tim77** [VirusTotal/qt-virustotal-uploader#14]
Would be nice to have universal Linux package which could installed without hassle. More info on [official site](https://flatpak.org/) and [Flathub](https://github.com/flathub/flathub#contributing-to-flathub). I'll try to help with initial build and submitting package on Flathub soon. If you interesting in maintain it let me know.

**Comment by nobaraos12** [comment]
project seems to be abandoned

**Comment by tim77** [comment]
Yep, but app is still works fine and [available](https://flathub.org/apps/details/com.virustotal.VirusTotalUploader) on Flathub. JFYI.

**Comment by LeandroStanger** [comment]
Please, consider supporting [Flatpak](http://docs.flatpak.org/en/latest/introduction.html#reasons-to-use-flatpak) and publishing **VirusTotal uploader** on [Flathub](https://github.com/flathub/flathub/wiki/App-Submission).

**Comment by noeedgarsuave1-hash** [comment]
Bgjh
